src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -453,7 +469,7 @@ private function createBlock(iterable $messages, string $type = null, string $st
 
         if (null !== $type) {
             $type = sprintf('[%s] ', $type);
-            $indentLength = \strlen($type);
+            $indentLength = Helper::width($type);
```

Comment:
It doesn't look directly related to your change no? Asking this in case as it feels like there is a missing test for it and I don't know if your new tests covers it or not

Reply:
One of my new tests covers it: https://github.com/symfony/symfony/pull/45318/files#diff-6ef90609877201be87f303aa59301e30a147c322dffba8f96f3e27655a5bcf0fR11